Doc/library/unittest.mock.rst

```diff
@@ -2615,7 +2615,7 @@ unit tests. Testing everything in isolation is all fine and dandy, but if you
 don't test how your units are "wired together" there is still lots of room
 for bugs that tests might have caught.
 
-:mod:`mock` already provides a feature to help with this, called speccing. If you
+:mod:`unittest.mock` already provides a feature to help with this, called speccing. If you
 use a class or instance as the :attr:`spec` for a mock then you can only access
```

Comment:
There is a warning on "spec" here.

Reply:
Yep, it's not yet documented, can you suggest some text for it? Also for `spec_set`?

Reply:
Maybe just write ```:attr:`!spec` ```.

Reply:
Done, and also for `spec_set`.